params/params_mixnet.go

```diff
@@ -41,6 +41,9 @@ var MixNetParams = Params{
 	ReduceMinDifficulty:  false,
 	MinDiffReductionTime: 0, // Does not apply since ReduceMinDifficulty false
 	GenerateSupported:    true,
+	UnlocksPerHeight:     10000 * 1e8, // every height 10000 MEER
```

Comment:
where are those global configure items (line 44:46) coming from? and looks there is no where they used.  Please remove if unnecessary. I wonder if the `params` is a good palace to set the lock/unlock rules.

Reply:
It useful in ledgerpayout.go on line 196